inc/namespace.php

```diff
@@ -124,10 +128,27 @@ function override_two_factor_universally_forced( bool $is_forced ) : bool {
  * @return array|null
  */
 function override_two_factor_forced_user_roles( $roles ) {
+	if ( maybe_skip_two_factor_forcing_for_local_env() ) {
+		return [];
+	}
+
 	$config = Altis\get_config()['modules']['security']['2-factor-authentication'];
 	if ( ! empty( $config['required'] ) && is_array( $config['required'] ) ) {
 		return $config['required'];
 	}
 
 	return $roles;
 }
+
+/**
+ * Override the two factor forcing filters for local environments.
+ *
+ * @return bool
+ */
+function maybe_skip_two_factor_forcing_for_local_env() : bool {
+	if ( Altis\get_environment_type() === 'local' ) {
+		return apply_filters( 'altis.security.2-factor-authentication.skip-local', true );
```

Comment:
Linter error here, I think we use the dot separators but still use underscores in filters and actions otherwise.
```suggestion
		/**
		 * Filters whether to skip forcing 2fa on local environments, defaults to true.
		 *
		 * @param bool $skip If true will skip forcing 2fa.
		 */
		return apply_filters( 'altis.security.2_factor_authentication.skip_local', true );
```